inst/workflow/data_mapping.yaml

```
@@ -1,141 +1,54 @@
-    output: Mapped_ImportantPD
+    output: Mapped_PD
     params:
       df: Raw_PD
-      strQuery: "SELECT subjectenrollmentnumber as subjid, * FROM df WHERE deemedimportant == 'Yes'"
+      strQuery: "SELECT subjectenrollmentnumber as subjid, deemedimportant FROM df"
   - name: RunQuery
-    output: Mapped_AllLabs
+    output: Mapped_LB
     params:
       df: Raw_LB
-      strQuery: "SELECT * FROM df WHERE toxgrg_nsv IN ('0', '1', '2', '3', '4')"
+      strQuery: "SELECT subjid, toxgrg_nsv FROM df"
   - name: RunQuery
-    output: Mapped_ToxLabs
-    params:
-      df: Raw_LB
-      strQuery: "SELECT * FROM df WHERE toxgrg_nsv IN ('3', '4')"
-  - name: RunQuery
-    output: Mapped_StudyDropouts
+    output: Mapped_STUDCOMP
     params:
       df: Raw_STUDCOMP
-      strQuery: "SELECT * FROM df WHERE compyn IN ('N')"
+      strQuery: "SELECT subjid, compyn FROM df"
   - name: RunQuery
-    output: Mapped_TreatmentDropouts
+    output: Mapped_SDRGCOMP
     params:
       df: Raw_SDRGCOMP
-      strQuery: "SELECT * FROM df WHERE sdrgyn IN ('N') AND phase = 'Blinded Study Drug Completion'"
+      strQuery: "SELECT subjid, sdrgyn, phase FROM df"
   - name: RunQuery
-    output: Mapped_ValidQueries
+    output: Mapped_QUERY
     params:
       df: Raw_QUERY
-      strQuery: "SELECT subjectname as subject_nsv, * FROM df WHERE querystatus IN ('Open', 'Answered', 'Closed')"
+      strQuery: "SELECT subjectname as subject_nsv, querystatus, queryage FROM df"
   - name: RunQuery
-    output: Mapped_OldValidQueries
-    params:
-      df: Mapped_ValidQueries
-      strQuery: "SELECT * FROM df WHERE queryage > 30"
-  - name: RunQuery
-    output: Mapped_DataChanges
+    output: Mapped_DATACHG
     params:
       df: Raw_DATACHG
-      strQuery: "SELECT subjectname as subject_nsv, * FROM df"
+      strQuery: "SELECT subjectname as subject_nsv, n_changes FROM df"
   - name: RunQuery
-    output: Mapped_Query
-    params:
-      df: Raw_QUERY
-      strQuery: "SELECT subjectname as subject_nsv, * FROM df"
-  - name: RunQuery
-    output: Mapped_DataEntry
-    params:
-      df: Raw_DATAENT
-      strQuery: "SELECT subjectname as subject_nsv, * FROM df"
-  - name: RunQuery
-    output: Mapped_SlowDataEntry
+    output: Mapped_DATAENT
     params:
       df: Raw_DATAENT
-      strQuery: "SELECT subjectname as subject_nsv, * FROM df WHERE data_entry_lag > 10"
-  - name: RunQuery
-    output: Mapped_ChangedDataPoints
-    params:
-      df: Raw_DATACHG
-      strQuery: "SELECT subjectname as subject_nsv, * FROM df WHERE n_changes > 0"
-  - name: RunQuery
-    output: Mapped_Screened
-    params:
-      df: Raw_ENROLL
-      strQuery: "SELECT * FROM df"
+      strQuery: "SELECT subjectname as subject_nsv, data_entry_lag FROM df"
   - name: RunQuery
-    output: Mapped_ScreenFail
+    output: Mapped_SCREEN
     params:
       df: Raw_ENROLL
-      strQuery: "SELECT * FROM df WHERE enrollyn = 'N'"
+      strQuery: "SELECT subjid, invid, country, enrollyn FROM df"
```

```suggestion
      strQuery: "SELECT subjid, invid, country, enrollyn FROM df"

```
Include a newline at the end of the file. Otherwise `readLines()` throws a warning. RStudio has a setting under Code > Saving, "Ensure that source files end with newline", pretty much entirely to avoid triggering that warning.